app/controllers/projects_controller.rb

```diff
@@ -92,9 +97,7 @@ def project_params
   end
 
   def project
-    @project ||= Project.find_by_param!(params[:id]).tap do |project|
-      project.current_user = current_user
-    end
+    @project
```

Comment:
After the refactoring following what we've discussed, I changed this method to return `current_project`